lib/engines/content_block_manager/app/models/content_block_manager/content_block/document/scopes/searchable_by_title.rb

```diff
@@ -0,0 +1,13 @@
+module ContentBlockManager
+  module ContentBlock::Document::Scopes::SearchableByTitle
+    extend ActiveSupport::Concern
+
+    included do
+      scope :with_title,
```

Comment:
I've adjusted this function in the optional last commit which diverges from Whitehall way of matching